src/UI/templates/default/Dropzone/tpl.standard-dropzone.html

```diff
@@ -0,0 +1,18 @@
+<div id="{ID}" class="il-dropzone-base">
+	<div class="clearfix hidden-sm-up"></div>
+	{FILELIST}
+	<div class="container-fluid">
+		<div class="il-dropzone standard clearfix row" data-upload-id="{ID}">
+			<div class="col-xs-12 col-md-9 col-sm-12 col-lg-9 col-no-padding">
+				<span class="pull-left dz-default dz-message">{MESSAGE}</span>
+			</div>
+			<div class="col-xs-12 col-md-3 col-sm-12 col-lg-3 il-dropzone-standard-select-files-wrapper text-right col-no-padding">
+				{SHY_BUTTON}
```

Comment:
There seems to be an issue here with the shy button overlapping the border if message is to long.